src/main/java/org/osiam/Osiam.java

```diff
@@ -2,20 +2,16 @@
 
 import com.google.common.collect.ImmutableMap;
 import com.ryantenney.metrics.spring.config.annotation.EnableMetrics;
-import com.zaxxer.hikari.HikariConfig;
-import com.zaxxer.hikari.HikariDataSource;
+import com.zaxxer.hikari.*;
```

Comment:
So I meant to introduce single class import as [coding convention](https://github.com/osiam/osiam/blob/master/CONTRIBUTING.md#coding-conventions).